### ISSUE TITLE
lr_config problem

### ISSUE THREAD

**Issue by xuhao-anhe** [open-mmlab/mmdetection#7781]
Thank you for your contribution. I'm a beginner of deep learning. When using the mmdetection training framework, I found a problem. I'm in LR_ Step = [28, 34] is set in config. It is found in many log files I run that when epoch = 27, the map decreases by a large margin compared with epoch = 26. What are the special requirements for the assignment of this step? The epoch of my whole training is only 50, and all the parameters used are given by the system, warmup_ iters = 500, so another question is whether only epoch is greater than warmup_ iters, the learning rate I set in optimizer will be used.

**Comment by xuhao-anhe** [comment]
> Thank you for your contribution. I'm a beginner of deep learning. When using the mmdetection training framework, I found a problem. I'm in LR_ Step = [28, 34] is set in config. It is found in many log files I run that when epoch = 27, the map decreases by a large margin compared with epoch = 26. What are the special requirements for the assignment of this step? The epoch of my whole training is only 50, and all the parameters used are given by the system, warmup_ iters = 500, so another question is whether only epoch is greater than warmup_ iters, the learning rate I set in optimizer will be used.
